### DISCUSSION TITLE
error Needs systemd

### DISCUSSION THREAD

**Discussion by ifenge** [containerd/nerdctl#1332] [Q&A] [answered]
hp pc,new installed debian 11,
run tar Cxzvvf /usr/local nerdctl-full-0.22.2-linux-amd64.tar.gz
run ./containerd-rootless-setuptool.sh install
[ERROR] Needs systemd (systemctl --user)
need help!

**Comment by fahedouch** [comment, marked as answer]
You need a systemd user instance to handle containerd  rootless user units.

**Reply by ifenge** [reply]
A new user has been created and su -l new user

**Reply by fahedouch** [reply]
Do you have systemd user instance for other user ? The problem is only with new user ? What gives
`systemctl --user status`

**Reply by ifenge** [reply]
new user run systemctl --user status, An error has occurred
Failed to connect to bus: $DBUS_SESSION_BUS_ADDRESS and $XDG_RUNTIME_DIR not defined (consider using --machine=<user>@.host --user to connect to bus of other user)

**Reply by ifenge** [reply]
I'm new,see  https://rootlesscontaine.rs/getting-started/common/ and Using google did not find a solution to the problem, hope you can give me a detailed operation process.thank you very much.

**Reply by ifenge** [reply]
Problem has been resolved.can't su new user, need to log in to the new user directly with ssh.Thank you for your help.nerdctl Running well.

**Comment by sullivan50909** [comment]
I had a similar issue. Based on this link: https://rootlesscontaine.rs/getting-started/common/login/ I tried using a new terminal window that I started using `ssh user@server` rather than` ssh root@server` and `su user`. When I ssh'd as user, I didn't get the `systemctl --user status `error.
